code/datums/supply_packs/medical.dm

```diff
@@ -66,52 +18,25 @@
 			/obj/item/bodybag/cryobag,
 			/obj/item/bodybag/cryobag
 			)
-	cost = RO_PRICE_NORMAL
+	cost = 40
 	containertype = /obj/structure/closet/crate/medical
 	containername = "stasis bag crate"
 	group = "Medical"
 
 /datum/supply_packs/surgery
-	name = "surgery crate (x2 surgical trays)"
+	name = "surgery crate (All the items to do surgery )"
```

Comment:
could use a more descriptive name, like "tray, anesthetic" or something rather than "all the items"

Reply:
done